uint/src/uint.rs

```diff
@@ -401,7 +401,7 @@ macro_rules! construct_uint {
 		/// Little-endian large integer type
 		#[repr(C)]
 		$(#[$attr])*
-		#[derive(Copy, Clone, Eq, PartialEq, Hash)]
+		#[derive(Copy, Clone, Eq, PartialEq, Hash, Serialize, Deserialize)]
```

Comment:
I think we should put this behind a feature.
So, only when this feature is activated, we add `Serialize`/`Deserialize` derives.

Reply:
Do we actually want it at all? In most cases we use hex-encoded version instead of just vector of numbers. The type is in the scope so it's super easy to implement the serialisation there (for instance using our `impl-serde` crate).

Reply:
Or that way, did not know that.